### ISSUE TITLE
Show modern macros in e.g. rpm-buildroot-usage

### ISSUE THREAD

**Issue by jengelh** [rpm-software-management/rpmlint#541]
```
[ 8319s] MozillaFirefox.src:540: W: rpm-buildroot-usage %build mkdir -p %{buildroot}%{progdir}/browser/extensions
[ 8319s] $RPM_BUILD_ROOT should not be touched during %build or %prep stage, as it may
[ 8319s] break short circuit builds.
```

(#540 would also apply here.)

Proposed new wording:

```
%buildroot ($RPM_BUILD_ROOT) should not be touched during %build or %prep stage, as it may break short circuit builds.
```

**Comment by scarabeusiv** [comment]
it should be fixed, otoh buildroot is now actively pruned by rpm 4.16 before install so you can touch it but it won't survive anymore at all.

**Comment by Conan-Kudo** [comment]
Yeah, I [fixed this in RPM 4.16](https://github.com/rpm-software-management/rpm/commit/f49671d6f3442cfff08d6167b618607e96cf0970), so this is probably a valid error.

**Comment by Conan-Kudo** [comment]
This warning should be upgraded to an error, since this is completely wrong.
